usecase/input/user.go

```diff
@@ -5,9 +5,9 @@ type GetUserDTO struct {
 }
 
 type CreateUserDTO struct {
-	LoginID    string `json:"loginID" validate:"required"`
-	Password   string `json:"password" validate:"required"`
-	FamilyName string `json:"familyName" validate:"required"`
-	GivenName  string `json:"givenName" validate:"required"`
-	RoleID     int    `json:"roleID" validate:"required"`
+	LoginID       string `json:"loginID" validate:"required"`
+	Password      string `json:"password" validate:"required"`
+	FamilyName    string `json:"familyName" validate:"required"`
+	GivenName     string `json:"givenName" validate:"required"`
+	IsShopManager bool   `json:"is_shop_manager" validate:"required"`
```

Comment:
[must]
キャメルケースでお願いします！
```suggestion
	IsShopManager bool   `json:"isShopManager" validate:"required"`
```